pkg/csi/manila/identityserver.go

```diff
@@ -40,7 +39,12 @@ func (ids *identityServer) GetPluginInfo(ctx context.Context, req *csi.GetPlugin
 }
 
 func (ids *identityServer) Probe(ctx context.Context, req *csi.ProbeRequest) (*csi.ProbeResponse, error) {
-	return &csi.ProbeResponse{}, nil
+	csiConn, err := ids.d.csiClientBuilder.NewConnectionWithContext(ctx, ids.d.fwdEndpoint)
+	if err != nil {
+		return nil, status.Error(codes.Unavailable, fmtGrpcConnError(ids.d.fwdEndpoint, err))
```

Comment:
this should be FAILED_PRECONDITION according to spec? https://github.com/container-storage-interface/spec/blob/master/spec.md#probe-errors

Reply:
@ramineni You're right, thanks! Fixed